### PR TITLE
Clipboard changed to community library. Warning.

### PR DESCRIPTION
**Pull request by juanky201271**
Nothing serious...

- change react native clipboard to community clipboard.

Screen shots:

Screenshot_20220427_082212
Screenshot_20220427_084933

### CODE REVIEW

**Comment by zancas** [comment]
I like the structure of a PR with a clear simple purpose.   Does this change fix a warning?

Unfortunately I don't know enough React-Native to review quickly.

Can we display the fixed warning somehow?
Screenshots, before and after?

**Comment by zancas** [comment]
I think we need @Gunnar-Stunnar to review.

**Comment by juanky201271** [comment]
I solved this one too, just in case.

File: components/PrivKeyModal.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable react-native/no-inline-styles */
 import React, {useState} from 'react';
-import {View, ScrollView, SafeAreaView, Image, Text, Platform, Clipboard, TouchableOpacity} from 'react-native';
+import {View, ScrollView, SafeAreaView, Image, Text, Platform, TouchableOpacity} from 'react-native';
+import Clipboard from '@react-native-community/clipboard';
```
Comment (zancas) [inline components/PrivKeyModal.tsx:4]: OK, so this `Clipboard` has a different implementation?

**Review by zancas — APPROVED** [review]

**Pull request merged**